test/test-search.ts

```diff
@@ -65,34 +64,44 @@ test('line number is accurate for search', async t => {
 });
 
 test(
-    'search function should return correct dynamic eval position with ' +
+    'search function should return correct dynamic arg position with ' +
         'require arg as a concatenation of strings that forms http',
     async t => {
       const content = 'const a = require(\'h\' + \'t\' + \'t\' + \'p\');';
       const result = await search.search(content);
-      t.deepEqual(result.dynamicEvals[0].lineStart, 1);
+      t.deepEqual(result.dynamicArgs[0].lineStart, 1);
     });
 
 test(
-    'search function should return correct dynamic eval position with ' +
+    'search function should return correct dynamic arg position with ' +
         'require arg as a substring that forms http',
     async t => {
       const content1 =
           `const a = \'anotherhttp\'\nconst b = require(a.substring(6));`;
       const result1 = await search.search(content1);
-      t.deepEqual(result1.dynamicEvals[0].lineStart, 2);
+      t.deepEqual(result1.dynamicArgs[0].lineStart, 2);
 
       const content2 = 'const a = require(\'anotherhttp\'.substring(6))';
       const result2 = await search.search(content2);
-      t.deepEqual(result2.dynamicEvals[0].lineStart, 1);
+      t.deepEqual(result2.dynamicArgs[0].lineStart, 1);
     });
 
 test(
-    'search function should return correct dynamic eval position with ' +
+    'search function should return correct dynamic arg position with ' +
         'require arg as a function that returns http',
     async t => {
       const content =
           'function returnHttp(){return \'http\';}\nconst a = require(returnHttp);';
       const result = await search.search(content);
-      t.deepEqual(result.dynamicEvals[0].lineStart, 2);
+      t.deepEqual(result.dynamicArgs[0].lineStart, 2);
+    });
+
+test(
```

Comment:
Are there other cases that are worth testing for re-assignment of require?

Reply:
Added some tests that we talked about earlier!